airflow/www/templates/airflow/dags.html

```diff
@@ -218,8 +218,9 @@ <h2>DAGs</h2>
       $('#dag_query').on('keypress', function (e) {
         // check for key press on ENTER (key code 13) to trigger the search
         if (e.which === ENTER_KEY_CODE) {
-          search_query = $('#dag_query').val();
-          window.location = DAGS_INDEX + "?search="+ encodeURI(search_query);
+          var query = new URLSearchParams(window.location.search);
```

Comment:
IE doesn't support this, so I think since it's easy to not use this we shouldn't.

(That or we add this in a polyfila)

Reply:
@ashb I checked the web interface in IE 11 before the changes and it did not work.
JS code has a syntax error (IE11) in other parts. [=> and \` ]

I added `url-search-params-polyfill` in separeted bransh [AIRFLOW-3360-polyfill](https://github.com/Doncode/airflow/tree/feature/AIRFLOW-3360-polyfill)  
Should i add it to PR?

Reply:
Oh right - I didn't realise it was that broken.

Still - it's an easy fix and it gets us to a better state, so yeah lets add that here and things get slowly better.

Reply:
@ashb please review